README.md

```diff
@@ -640,6 +640,11 @@ Described [here](https://chris.beams.io/posts/git-commit/).
 There are many such conventions, but this is the one we use.
 It's pretty straightforward.
 
+#### git rebase
```

Comment:
maybe to avoid silly questions, it's best to clarify "by rebasing, we mean any kind of tree mutation, anything that would require a force push"?